.github/workflows/product-is-builder-jdk21.yml

```diff
@@ -0,0 +1,59 @@
+name: product-is-builder-jdk21
+
+on:
+  workflow_dispatch:
+  schedule:
+#    At 00:00 on day-of-month 1
+    - cron:  '0 0 1 * *'
+
+
+env:
+  MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.class=standard -Dmaven.wagon.http.retryHandler.count=3
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      JAVA_TOOL_OPTIONS: "-Djdk.util.zip.disableZip64ExtraFieldValidation=true -Djdk.nio.zipfs.allowDotZipEntry=true"
+
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v2
+      - name: Set up Adopt JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: "11"
+          distribution: "adopt"
+      - name: Product-IS build with JDK 11 without Integration Tests
+        run: |
+          mvn clean install --batch-mode -Dmaven.test.skip=true | tee mvn-build.log
```

Comment:
Why do we run integration tests with both JDK 11 and 21?

Reply:
-Dmaven.test.skip=true will skip the tests. We are building the pack on JDK 11 in this step. Next step is running the only the integration tests on JDK 21